src/Domain/MetaCampaignRepository.php

```diff
@@ -45,4 +46,74 @@ public function countCompleteDonationsToMetaCampaign(MetaCampaign $metaCampaign)
 
         return $count;
     }
+
+    /**
+     * Returns the total of all the complete donations to this metacampaign, including matching and including gift aid,
+     * and any "offline" donations or adjustments.
+     */
+    public function totalAmountRaised(MetaCampaign $metaCampaign): Money
+    {
+        $donationQuery = $this->em->createQuery(
+            <<<'DQL'
+            SELECT donation.currencyCode, COALESCE(SUM(
+            donation.amount + 
+            if(donation.giftAid, donation.amount * 0.25, 0)
```

Comment:
Could use `Donation::GIFT_AID_PERCENTAGE`?

And/or, wondering if there's possibly an argument that a database level view with a `giftAidAmount` could be helpful here, to:
1. reduce on-the-fly calculation generally, and
2. make it less painful to denormalise differently should the Gift Aid rate change